src/bidiTab/mapperTabPage.ts

```diff
@@ -69,8 +69,11 @@ export function log(logType: LogPrefix, ...messages: unknown[]) {
     return;
   }
 
-  // If `sendDebugMessage` is defined, send the log message there.
-  global.window?.sendDebugMessage?.(JSON.stringify({logType, messages}));
+  // Skip sending BiDi logs as they are logged once by `bidi:server:*`
+  if (!logType.startsWith(LogType.bidi)) {
```

Comment:
Move to here as it makes more sense to not send the info back rather then sending and filtering on the other end.